reel-picker/src/main/java/com/innowisegroup/reelpicker/picker/ui/DatePickerFragment.kt

```diff
@@ -32,19 +32,13 @@ internal class DatePickerFragment : Fragment() {
         savedInstanceState: Bundle?
     ): View? {
         val view = inflater.inflate(R.layout.item_date_picker, container, false)
+        applyArguments()
```

Comment:
Why didn't you use the same structure in all fragments
        applyArguments()
        applyAttributes()
        setFragmentResultListeners()
        initializeView(view)

Reply:
Also can add applySavedStateIfNeed(savedInstanceState) where you can put all code 
if (savedInstanceState != null) {......}